src/main/java/org/jabref/model/entry/ParsedFileField.java

```diff
@@ -1,7 +1,14 @@
 package org.jabref.model.entry;
 
+import java.io.File;
 import java.net.URL;
+import java.nio.file.Path;
 import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.logic.util.io.FileUtil;
```

Comment:
I moved the method to FileUtil as you proposed and will open a new PR with the refactoring since it turned out to be a bit more extensive than I have expected.